boa3/model/builtin/interop/contract/createmethod.py

```diff
@@ -14,16 +14,10 @@ def __init__(self, contract_type: ContractType):
         syscall = 'deploy'
         args: Dict[str, Variable] = {
             'nef_file': Variable(Type.bytes),
-            'manifest': Variable(Type.bytes)
+            'manifest': Variable(Type.bytes),
+            'data': Variable(Type.any)
         }
+        data_default = ast.parse("{0}".format(Type.sequence.default_value)
```

Comment:
`Type.sequence.default_value` is `[]`, but you defined the default value of `data` to `None` in the `builtin` package